robot/camera.py

```diff
@@ -61,7 +104,7 @@ def _cam_listener_worker(self):
                     self._latest = data
 
     @staticmethod
-    def _see_to_results(data):
+    def _see_to_results(data) -> ResultList:
```

Comment:
Please can we also type `data`

Reply:
I think building an accurate type-based schema for the results of `robotd` would be great, but maybe OOS for this PR. Doing something like `Dict[str, Any]` is accurate, but not especially useful. 

I'm not exactly sure if you can specify keys for dictionaries yet, I have a slight feeling you can't? If not then perhaps I can just use a standard `JSON` type, from https://github.com/python/typing/issues/182